app/views/discussions/print.html.haml

```diff
@@ -0,0 +1,101 @@
+          - if motion == @discussion.current_motion
+            %h3= t :current_decision
+          %h3= motion.title
+          .proposal-description
+            = motion.description
+          .proposal-details
+            %ul
+              - if motion.voting?
+                %li= t(:closing) + ": " + motion.closing_at.strftime("%d %B, %I.%M %p %Z")
+              - else
+                %li= t(:closed_at) + ": " + motion.closed_at.strftime("%d %B, %I.%M %p %Z")
+              %li= t :proposed_by_who_on, when: motion.created_at.strftime("%d %B, %I.%M %p %Z"), who: motion.author_name
+            - if motion.has_outcome?
+              %strong= t(:outcome) + ": "
+              %span
+                = motion.outcome
+        .col-md-9
+          .motion-graph.clearfix
+            %canvas.pie{id: "graph_#{motion.id}", height: 150, width: 150}
+            .legend
+              .item.yes
+                = t(:'position_verbs.yes')
+                (#{motion.yes_votes_count})
+              .item.abstain
+                = t(:'position_verbs.abstain')
+                (#{motion.abstain_votes_count})
+              .item.no
+                = t(:'position_verbs.no')
+                (#{motion.no_votes_count})
+              .item.block
+                = t(:'position_verbs.block')
+                (#{motion.block_votes_count})
+
+              - if motion.closed?
+                = t :engagement_on_closed_proposal,
+                    percent: motion.percent_voted,
+                    numerator: (motion.group_size_when_voting - motion.members_not_voted_count),
+                    denominator:  motion.group_size_when_voting
+              - else
+                = t :engagement_on_open_proposal,
+                    percent: motion.percent_voted,
+                    numerator: (motion.group_size_when_voting - motion.members_not_voted_count),
+                    denominator:  motion.group_size_when_voting
+            = render('motions/motion_pie_javascript', motion: motion)
+        .col-md-9
+          %h3= t :positions
+          %ul.positions
+            - if motion.has_votes?
+              - motion.grouped_unique_votes.each do |vote|
+                - icon = display_proposal_activity_icons_for_print(vote)
+                %li
+                  = image_tag icon
+                  %span
+                    %strong= vote.user_name
+                  %span= vote.statement
+  - if @activity
+    .row
+      .col-md-9
+        %h3= t :activity
+        %ul.activity
+          - filter_discussion_events(@activity).each do |event|
+            - if event.kind == 'new_comment'
+              %li
+                = render 'avatar', user: event.eventable.user
+                %span
+                  %strong= event.eventable.user_name
+                %span= event.eventable.body
+                %span.timestamp= event.eventable.created_at.strftime("%d %B, %I.%M %p")
+            - else
+              - item = DiscussionItem.new(event)
```

Comment:
I'm a bit confused by this bit; what's going on here? Seems pretty scary be initializing DiscussionItems in the view like this.